<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcRegister</class>
 <widget class="QMainWindow" name="OcRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OC Register</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/downloads/newicons/clock_red.png</normaloff>:/downloads/newicons/clock_red.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkSpecificCustomer">
        <property name="text">
         <string>Sepecific Customer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dtpStartDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="cboCustomers"/>
      </item>
      <item row="0" column="4">
       <widget class="QDateEdit" name="dtpEndDate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkIsRange">
        <property name="text">
         <string>Is Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="cmdPrint">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="cmdShowOcs">
        <property name="text">
         <string>Show OCs</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>OC Register</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CustomQueryView" name="lstOcs"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tonnage History</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCustomPlot" name="tonnageHistoryGraph" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkIsRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtpEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSpecificCustomer</sender>
   <signal>toggled(bool)</signal>
   <receiver>cboCustomers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
